samples/FunctionApp/HttpProxyTrigger/HttpProxyTrigger.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Threading;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+
+namespace FunctionApp
+{
+    public class HttpProxyTrigger
+    {
+        [Function(nameof(HttpProxyTrigger))]
```

Comment:
Outside of scope for this PR, but do we want to make a different sample app for the aspnet integration examples? Would make sense since the wire up is different + we don't want to interfere with the current http model being used in this app.

Reply:
We should